CHANGELOG.md

```diff
@@ -14,31 +16,56 @@ might need to be adjusted and either explicitly name the positional
 argument or resolve the type ambiguity by casting to the intended type.
 
 **Features**
+* PG15 support added
+* MultiNode High Avalibility improvements
+* #4335 Show warnings when not following best practices
+* #4476 Batch rows on access node for distributed COPY
+* #4562 Exponentially backoff when out of background workers
 * #4650 Show warnings when not following best practices
+* #4664 Introduce fixed schedules for background jobs
+* #4668 Hierarchical Continuous Aggregates
 * #4670 Add timezone support to time_bucket_gapfill
+* #4678 Add interface for troubleshooting job failures
 * #4718 Add ability to merge chunks while compressing
 * #4786 Extend the now() optimization to also apply to CURRENT_TIMESTAMP
+* #4820 Support parameterized data node scans in joins
+* #4830 Add function to alter data nodes
 
 **Bugfixes**
+* #4663 Don't error when compression metadata is missing
 * #4673 Fix now() constification for VIEWs
 * #4681 Fix compression_chunk_size primary key
 * #4696 Report warning when enabling compression on hypertable
+* #4733 Fix an ereport syntax error when using PG 12.0
 * #4745 Fix FK constraint violation error while insert into hypertable which references partitioned table
 * #4756 Improve compression job IO performance
+* #4770 Continue compressing other chunks after an error
+* #4794 Fix degraded performance seen on timescaledb_internal.hypertable_local_size() function
 * #4807 Fix segmentation fault during INSERT into compressed hypertable
-* #4840 Fix performance regressions in the copy code
+* #4809 Fix ereport call in dist_copy for PG 12.0
```

Comment:
Not a bug, this is a fix for a change that was never part of a released version.
```suggestion
```

Reply:
Removed